tests/integration/document_db.py

```diff
@@ -0,0 +1,205 @@
+    assets = response.assets_updated(asset_type=asset_type)
+    assert len(assets) == 1
+
+class TestConnection:
+    connection: Optional[Connection] = None
+
+    def test_create(
+        self,
+        client: AtlanClient,
+        upsert: Callable[[Asset], AssetMutationResponse],
+    ):
+        role = client.role_cache.get_id_for_name("$admin")
+        assert role
+        connection_name = TestId.make_unique("DOC_Conn")
+        c = Connection.create(
+            name=connection_name,
+            connector_type=AtlanConnectorType.DOCUMENTDB,
+            admin_roles=[role],
+        )
+        assert c.guid
+        response = upsert(c)
+        assert response.mutated_entities
+        assert response.mutated_entities.CREATE
+        assert len(response.mutated_entities.CREATE) == 1
+        assert isinstance(response.mutated_entities.CREATE[0], Connection)
+        assert response.guid_assignments
+        assert c.guid in response.guid_assignments
+        c = response.mutated_entities.CREATE[0]
+        c = client.asset.get_by_guid(c.guid, Connection, ignore_relationships=False)
+        assert isinstance(c, Connection)
+        TestConnection.connection = c
+
+    @pytest.mark.order(after="test_create")
+    def test_trim_to_required(
+        self, client: AtlanClient, upsert: Callable[[Asset], AssetMutationResponse]
+    ):
+        assert TestConnection.connection
+        connection = TestConnection.connection.trim_to_required()
+        response = upsert(connection)
+        assert response.mutated_entities is None
+
+
+@pytest.mark.order(after="TestConnection")
+class TestDatabase:
+    database: Optional[DocumentDBDatabase] = None
+
+    def test_create(
+        self,
+        client: AtlanClient,
+        upsert: Callable[[Asset], AssetMutationResponse],
+    ):
+        assert TestConnection.connection
+        connection = TestConnection.connection
+        assert connection
+        assert connection.qualified_name
+        database_name = TestId.make_unique("DocDB")
+        database = DocumentDBDatabase.creator(
+            name=database_name,
+            connection_qualified_name=connection.qualified_name,
+        )
+        assert database.guid
+        response = upsert(database)
+        assert response.mutated_entities
+        assert response.mutated_entities.CREATE
+        assert connection.qualified_name == database.connection_qualified_name
+        assert len(response.mutated_entities.CREATE) == 1
+        assert isinstance(response.mutated_entities.CREATE[0], DocumentDBDatabase)
+        assert response.guid_assignments
+        assert database.guid in response.guid_assignments
+        database = response.mutated_entities.CREATE[0]
+        client.asset.get_by_guid(database.guid, DocumentDBDatabase, ignore_relationships=False)
+        TestDatabase.database = database
+
+    @pytest.mark.order(after="test_create")
+    def test_create_for_modification(
```

Comment:
Just a heads-up - it’s now renamed to `updater()`
```suggestion
    def test_updater(
```